src/modals/mod-application.js

```diff
@@ -81,9 +81,9 @@ async function modApplicationHandler(interaction) {
 
 	const modApplicationEmbed = new Discord.EmbedBuilder();
 
-	modApplicationEmbed.setColor(0x0096FF);
+	modApplicationEmbed.setColor(0x9D6FF3);
```

Comment:
Why was this color changed?

Reply:
This color was changed to match the accent on the `Pending` banner Bandwidth sends with this message.